src/main/java/com/github/manolo8/darkbot/Main.java

```diff
@@ -153,7 +153,7 @@ private void validTick() {
         statsManager.tick();
 
         tickingModule = running && guiManager.canTickModule();
-        if (running && guiManager.canTickModule()) tickRunning();
+        if (tickingModule) tickRunning(); // should double check canTickModule()?
```

Comment:
```suggestion
        if (tickingModule) tickRunning();
```
Nice catch, it was probs like this from before behaviours